### PR TITLE
Add OpenSSL to compile options

### PR DESCRIPTION
**Pull request by aserdean**
This patch adds OpenSSL to the compile options of OVS.

Signed-off-by: Alin Gabriel Serdean aserdean@cloudbasesolutions.com

### CODE REVIEW

File: BuildAutomation/BuildOpenvSwitch.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Param(
-  [string]$OVSGitBranch = "branch-2.5-cloudbase",
-  [string]$SignX509Thumbprint
-)
+Param(
+  [string]$OVSGitBranch = "branch-2.5-cloudbase",
```
Comment (alexpilotti) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: This commit has quite a few unrelated whitespace changes

Comment (aserdean) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: ^M replaced from my editor
\t as well
I brought it to the defaults.

Comment (alexpilotti) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: Should have been a separate commit. Now it's difficult to revert if needed.

Comment (alexpilotti) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: merged anyway, but please keep in mind for future ones!

Comment (aserdean) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: we could always revert commit: https://github.com/cloudbase/openvswitch-hyperv-installer/commit/4d2bbd24646cc1f73edb0e586defecc621e45d00 (which introduced them) and fix white-spaces and do another pull request.

Comment (alexpilotti) [inline BuildAutomation/BuildOpenvSwitch.ps1:2]: Let's add a .gitattributes txt directive so we can enforce an EOL policy
